### PR TITLE
feat: revised PR (wiring different buttons and design review changes …

### PR DESCRIPTION
**Pull request by tiarascoleman**
…[VUE 1169, 1170, 1172, 1197])

Fixed PR comments!

### CODE REVIEW

**Review by dyersituations — APPROVED** [review]

**Comment by dyersituations** [comment]
Were these changes already merged in the previous PR? I see you have conflicts.

File: src/components/Settings/SaveSearchItem.vue [hunk 1]
```diff
@@ -46,6 +46,7 @@ export default {
 	name: 'SaveSearchItem',
 	data() {
 		return {
+
 			showAlerts: this.savedSearch?.isAlert,
 			deleteSearch: () => []
```
Comment (mcstover) [inline src/components/Settings/SaveSearchItem.vue:51]: Please remove the extra blank line and this `deleteSearch` key/value pair. It's unused.

File: src/components/Settings/SaveSearchItem.vue [hunk 2]
```diff
@@ -93,6 +94,7 @@ export default {
 					id: this.savedSearch?.id,
 					savedSearch: { isAlert }
 				}
+
```
Comment (mcstover) [inline src/components/Settings/SaveSearchItem.vue:97]: empty line...

File: src/pages/Settings/SavedSearchBeta.vue [hunk 3]
```diff
@@ -80,9 +80,11 @@ export default {
 				this.savedSearches = [...savedSearchData?.values];
 			});
 		},
+
 		deleteSavedSearch(payload) {
 			console.log(payload);
 		},
+
```
Comment (mcstover) [inline src/pages/Settings/SavedSearchBeta.vue:87]: Please remove the empty lines this method...

**Review by mcstover — APPROVED** [review]
There it is!

**Pull request merged**